g0v/vaccinate
# Add support for building out government-paid vaccines.

# Context
We should add support for searching for and booking vaccine appointments.

# This PR: 
1. Modifies the Python types to take in two different types of AppointmentAvailability.
2. Feeds this into the Redis database. 
3. Reads it from the Redis database and displays to the user. We've added two different radio buttons to let the user choose between self-paid and government-paid vaccinations. 

# Test Plan: 
Since we're modifying the infra here, we need to test that reading from redis and local scraping works correctly. 
1. Confirm `cd backend; pipenv run python local_scraper.py` runs without error.
2. Confirm `cd backend; pipenv run python app.py` runs without error.
3. Confirm `cd backend; yarn backend` runs without error.
